README.md

```diff
@@ -86,17 +99,11 @@ __kernel void mapkernel_5672850724456951104(__global const _1test *f, __global f
     c[gid - 1] = _ssavalue_0;
     ;
 }
-
-```
-
-If you want to run this example, you need to add and checkout the following packages:
-```Julia
-Pkg.clone("https://github.com/SimonDanisch/Transpiler.jl.git")
 ```
 
 # TODO / Common issues
 
 * compiling constructor code
-* Not sure how to transpile Core._apply
+* Not sure how to transpile `Core._apply`
```

Comment:
If there's `Core._apply` in the code after inference, that means dynamic dispatch, so that can't be transpiled (easily).

Reply:
Hm, this is actually old and thought I solved this.... but it's possible, that I solved it by changing the semantic